Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -98,6 +98,14 @@ namespace PhysX
             }
         }
 
+        /**
+        * Function to delete the ground for tests where the ground can interfere with the intent of the test. 
+        */
```

Comment:
nit: Please prefer `//!` style comments now (multi-line ones are harder to maintain and more fiddly to write)

Reply:
changed